tier4_system_msgs/msg/ElectionStatus.msg

```diff
@@ -0,0 +1,19 @@
+# leader_id
+uint8 MAIN_ECU = 0
+uint8 SUB_ECU = 1
+uint8 MAIN_VCU = 2
+uint8 SUB_VCU = 3
+
+builtin_interfaces/Time stamp
+uint8 leader_id
+uint8 path_info
```

Comment:
Are there constants for path info?

Reply:
If we were to express it, it would look like the following:
Main_ECU-Main_VCU = 10 (0x1010)
Main_ECU-Sub_VCU = 9 (0x1001),,,

I have included an explanation [69001df](https://github.com/tier4/tier4_autoware_msgs/pull/134/commits/69001df63759e16a4eb4b499862b1d1d1e224901), but would it be better to define constants?